pkg/runctl/run_manager.go

```diff
@@ -506,7 +506,10 @@ func (c *runManager) createTektonTaskRun(ctx *runContext) error {
 		},
 	}
 	c.addTektonTaskRunParamsForJenkinsfileRunnerImage(ctx, &tektonTaskRun)
-	c.addTektonTaskRunParamsForPipeline(ctx, &tektonTaskRun)
+	err = c.addTektonTaskRunParamsForPipeline(ctx, &tektonTaskRun)
+	if err != nil {
+		return serrors.Classify(err, stewardv1alpha1.ResultErrorInfra)
```

Comment:
Should it be classified as infrastructure problem or config problem?

Reply:
I was not sure about it. Is it the end customer causing the issue or is it the front end part of the service? But in both cases from back end perspective it is a config issue... you're right.
```suggestion
		return serrors.Classify(err, stewardv1alpha1.ResultErrorConfig)
```